chapter2/tests/t19/Makefile

```diff
@@ -0,0 +1,140 @@
+.PHONY: all _dev _staging _prod
+
+.DEFAULT_GOAL := help
+
+# ----------------------------------------------------------------------------
+# Local Variables
+#
+# ============================================================================
+include .makerc
```

Comment:
> This doesn't work correctly because you didn't add .makerc file

@kseniyashaydurova 
I just had a doubt whether I should push the file with IP addresses
Done